### ISSUE TITLE
Upgrade to Angular 14

### ISSUE THREAD

**Issue by AshleyMedway** [DethAriel/ng-recaptcha#262]
### Summary

I'm submitting a: 

- [ ] bug report
- [x] feature request
- [ ] question / support request 
- [ ] other

### Description

Add support for Angular 14

**Comment by lagranjadigital** [comment]
Please add support!

**Comment by AshleyMedway** [comment]
For those waiting for this I have published a separate package for support for Angular14 until the package owner can merge this in.

Remove the reference to `ng-recaptcha` then install `ng-recaptcha14`.

```
npm i ng-recaptcha14
```

You will also need to update any imports. i.e.

```
import { RecaptchaFormsModule, RecaptchaModule } from "ng-recaptcha14";
```

**Comment by Mashsquare** [comment]
In the end I managed to make the current version run, however it has to be said that the recaptcha script import should be done in the head of the index.htm and not in the html file of the component in which you are using it (for example a form) or it will work only in incognito

**Comment by DethAriel** [comment]
Hey folks! Apologies for the delay, but I wanted to let you know that support for Angular 14 has landed in `ng-recaptcha@10.0.0`. Cheers!
